<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPlainTextEdit" name="plainTextEdit_1">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="lineWrapMode">
       <enum>QPlainTextEdit::NoWrap</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>507</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="atn_new"/>
    <addaction name="atn_open"/>
    <addaction name="separator"/>
    <addaction name="atn_save_as"/>
    <addaction name="separator"/>
    <addaction name="atn_to_close"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Вид</string>
    </property>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="atn_new">
   <property name="text">
    <string>Новый...</string>
   </property>
  </action>
  <action name="atn_open">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="atn_save_as">
   <property name="text">
    <string>Сохранить как...</string>
   </property>
  </action>
  <action name="atn_to_close">
   <property name="text">
    <string>Закрыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
